packages/forms-web-app/src/pages/projects/section-51/utils/adviceVM.js

```diff
@@ -0,0 +1,41 @@
+const getAdviceName = ({ organisation, firstName, lastName }) => {
```

Comment:
I'd prob say a ViewModel was not a util - but at the same level as controller etc

Reply:
good shout. Have lifted it out